packages/react-heartwood-components/package.json

```diff
@@ -1,161 +1,162 @@
-		"@storybook/addon-links": "^5.1.10",
-		"@storybook/addon-viewport": "^5.1.10",
-		"@storybook/addons": "^5.1.10",
-		"@storybook/react": "^5.1.10",
-		"@types/react": "^16.8.7",
-		"awesome-typescript-loader": "^5.2.1",
-		"babel-core": "7.0.0-bridge.0",
-		"babel-eslint": "8",
-		"babel-loader": "^8.0.6",
-		"babel-plugin-inline-react-svg": "^0.5.4",
-		"babel-preset-react-app": "^9.0.0",
-		"babel-runtime": "^6.26.0",
-		"chokidar-cli": "^1.2.0",
-		"enzyme": "^3.3.0",
-		"enzyme-adapter-react-16": "^1.1.1",
-		"enzyme-to-json": "^3.3.1",
-		"eslint": "^5.9.0",
-		"eslint-config-spruce": "^8.17.7",
-		"faker": "^4.1.0",
-		"husky": "^0.14.3",
-		"jest": "^24.5.0",
-		"prettier": "1.16.4",
-		"react-docgen-typescript-loader": "^3.1.1",
-		"react-test-renderer": "^16.0.0",
-		"storybook-addon-jsx": "^7.1.5",
-		"storybook-addon-react-docgen": "^1.2.4",
-		"storybook-readme": "^5.0.6",
-		"typescript": "~3.4.5",
-		"uuid": "^3.3.2",
-		"webpack": "^4.29.6"
-	},
-	"dependencies": {
-		"@babel/polyfill": "^7.0.0",
-		"@babel/register": "^7.0.0",
-		"@sprucelabs/heartwood-components": "^8.17.7",
-		"@types/react-dom": "^16.8.2",
-		"@types/react-table": "^6.8.4",
-		"axios": "^0.18.0",
-		"classnames": "^2.2.5",
-		"cookies": "^0.7.1",
-		"debug": "^3.1.0",
-		"es6-tween": "^5.3.1",
-		"exenv": "^1.2.2",
-		"exif-orientation-image": "^1.0.1",
-		"fontfaceobserver": "^2.1.0",
-		"formik": "^1.4.2",
-		"image-webpack-loader": "^4.3.1",
-		"imagesloaded": "^4.1.4",
-		"is_js": "^0.9.0",
-		"js-cookies": "^1.0.4",
-		"jsdom": "14.0.0",
-		"jsdom-global": "3.0.2",
-		"memoize-one": "^4.0.2",
-		"moment": "^2.22.1",
-		"moment-timezone": "^0.5.21",
-		"next": "^7.0.0",
-		"next-redux-wrapper": "^1.0.0",
-		"prop-types": "^15.6.0",
-		"qs": "^6.5.1",
-		"rc-time-picker": "^3.3.1",
-		"react": "^16.3.2",
-		"react-autosuggest": "^9.4.2",
-		"react-avatar-editor": "^11.0.4",
-		"react-big-calendar": "0.19.2",
-		"react-dates": "^17.0.0",
-		"react-dnd": "2.6.0",
-		"react-dnd-html5-backend": "2.6.0",
-		"react-dnd-touch-backend": "^0.4.0",
-		"react-dom": "^16.3.2",
-		"react-dropzone": "^6.0.2",
-		"react-helmet": "^5.2.0",
-		"react-image-crop": "3.0.11",
-		"react-markdown": "^4.1.0",
-		"react-modal": "^3.6.1",
-		"react-phone-number-input": "^2.3.7",
-		"react-redux": "^5.0.6",
-		"react-required-if": "^1.0.3",
-		"react-sortable-hoc": "^0.8.3",
-		"react-table": "^6.8.6",
-		"react-textarea-autosize": "^6.0.0",
-		"react-transition-group": "^2.5.2",
-		"react-virtualized": "^9.21.0",
-		"react-with-direction": "^1.1.0",
-		"redux": "^3.7.2",
-		"redux-form": "^7.3.0",
-		"screenfull": "^3.3.3",
-		"velocity-react": "^1.4.1",
-		"webfontloader": "^1.6.28"
-	},
-	"jest": {
-		"roots": [
-			"<rootDir>/src/"
-		],
-		"coverageDirectory": "./coverage/",
-		"collectCoverage": true,
-		"coveragePathIgnorePatterns": [
-			"<rootDir>/lib/",
-			"<rootDir>/node_modules/"
-		]
-	}
+  "name": "@sprucelabs/react-heartwood-components",
```

Comment:
Looks like this file was saved without `node_modules` installed since it's now using spaces